test/lib/extractor/image.test.ts

```diff
@@ -155,7 +146,7 @@ test("oci image extractor: extracted image content returned as expected", async
     },
   ];
 
-  const result = await extractOciArchive(
+  const result = await ociExtractor.extractArchive(
```

Comment:
Wondering if we should use `getArchiveLayersAndManifest()` here with `ImageType.OciArchive`?
I think that it may be best to test the most top-level function rather than `extractArchive` which is a bit "internal" 🤷

Reply:
Yup, I agree, as this file are actually a higher level tests 👌